recipe/meta.yaml

```diff
@@ -54,23 +55,56 @@ outputs:
         - perl  # [osx]
         - autoconf  # [osx]
         - automake  # [osx]
-        - libtool   # [unix]
+        - binutils  # [linux]
+        - libevent
+        - libhwloc
+        - libmagic
+        - libnl  # [linux]
+        - libnuma  # [linux]
+        - libtool  # [unix]
+        - libudev1  # [linux]
         - make  # [unix]
+        - numactl  # [linux]
+        - pkg-config 
+        - rdma-core  # [linux]
+        - ucc  # [linux and not osx and not ppc64le]
+        - ucx  # [linux and not osx]
+        - zlib
+          
       host:
         #- openpmix
         #- prrte
-        - libhwloc
+        - binutils  # [linux]
         - libevent
+        - libhwloc
+        - libmagic
         - libnl  # [linux]
+        - libnuma  # [linux]
+        - libudev1  # [linux]
+        - numactl  # [linux]
```

Comment:
Are all of these new dependencies that are linked directly in order for this feature to work? That seems like a lot of direct dependencies to add.